templates/vue/cypress/integration/node-authoring.spec.js

```diff
@@ -235,4 +238,32 @@ describe("Node Authoring", () => {
       })
     })
   })
+
+  describe("As subscriber:", () => {
+    beforeEach(() => {
+      cy.setup("@subscriberNode", roles.SUBSCRIBER)
+    })
+
+    it("should be able to add a child node using the node modal as subscriber", () => {
+      cy.getSelectedNode().then(parent => {
+        const child = {
+          title: "Child 1",
+          mediaType: "text",
+          typeData: {
+            textContent: "Abcd",
+          },
+        }
+
+        cy.openModal("add", parent.id)
+        cy.getByTestId(`node-title`).type(child.title)
+
+        cy.changeMediaType(child.mediaType)
+        cy.getEditable(`node-text-content`).type(child.typeData.textContent)
+
+        cy.submitModal()
+        cy.contains(child.title).should("exist")
+        cy.get(".links").children().should("exist")
```

Comment:
I think a better assertion for this would be to check if the link between the child and the parent exists. You can use the `cy.link` command to get the link given the parent id and child id, and the `cy.getNodeByTitle` command to get the id of the child node. Something like:

```suggestion
        cy.getNodeByTitle(child.title).its("id").then(childId => {
          cy.link(parent.id, childId).should("exist")
        })
```